src/renderer/components/dialog/OpdsFeedAddForm.tsx

```diff
@@ -20,54 +20,94 @@ interface Props extends TranslatorProps {
 interface State {
     name: string;
     url: string;
+    opdsFeedList: {
+        [name: string]: string;
+    } | undefined;
+    selectValue: string;
 }
 
 export class OpdsFeedAddForm extends React.Component<Props, State> {
-    constructor(props: any) {
+    private defaultSelectValue = "undefined";
+    private opdsFeedListServerUrl = "http://thorium-opds.edrlab.org/";
```

Comment:
I would strongly suggest using the secure HTTPS URL (there is no justification for using HTTP nowadays).
(for example, if the renderer process is moved to secure HTTPS in the future, then this non-HTTPS `window.fetch` request will fail)
https://www.eff.org/https-everywhere
:)

Reply:
https://thorium-opds.edrlab.org/ doesn't works. spoken with Laurent

Reply:
The HTTP CORS headers are missing in the root EDRLab URL, so `fetch()` Web API will reject the request.
Workaround: use `request` NodeJS API instead of `window.fetch` (`nodeIntegration` will be removed *at some point*, but we can still use it now).

Reply:
Also note the discrepancy in the context where the HTTP request is created (i.e. renderer processes for library and reader BrowserWindows):
* dev: `http://localhost:8090` (WebPack dev server)
* prod: `file://` (normal Electron URL from the filesystem / ASAR bundle in a packaged app)